src/main/scala/it/pagopa/interop/apigateway/service/impl/CatalogProcessServiceImpl.scala

```diff
@@ -29,16 +29,23 @@ class CatalogProcessServiceImpl(invoker: CatalogProcessInvoker, api: EServiceApi
       .recoverWith { case err: ApiError[_] if err.code == 404 => Future.failed(EServiceNotFound(eServiceId)) }
   } yield result
 
-  override def getEServices(producerId: UUID, attributeId: UUID, offset: Int, limit: Int)(implicit
-    contexts: Seq[(String, String)]
-  ): Future[EServices] = for {
+  override def getEServices(
+    name: Option[String] = None,
+    eServicesIds: Seq[UUID],
+    producersIds: Seq[UUID],
+    attributesIds: Seq[UUID],
+    agreementStates: Seq[AgreementState],
+    states: Seq[EServiceDescriptorState],
```

Comment:
added parameters are not used in the call.
Are they useful? We can ignore them for now if not required

Reply:
> added parameters are not used in the call. Are they useful? We can ignore them for now if not required

I see, I forgot that they have a default and be omitted when not needed